src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BallLimit = 100;
```

Comment:
lets check constant naming

Reply:
I don`t understand what I should do here?